storm-core/src/clj/org/apache/storm/testing.clj

```diff
@@ -667,23 +665,18 @@
           (.put "spout-emitted" (AtomicInteger. 0))
           (.put "transferred" (AtomicInteger. 0))
           (.put "processed" (AtomicInteger. 0))))
+      (LocalExecutor/setTrackId id#)
       (with-var-roots
```

Comment:
We don't need with-var-roots any more, we could just call with simulated-time-local-cluster without it